chihacknight/decarbonize-my-state
# Favicons

## Overview

Adds favicons. Woo!

Closes part of #73

## Testing Instructions

- View site in browser
- See header
